docs/ReleaseHistory.md

```diff
@@ -134,7 +135,7 @@
 # 1.9.1 - 11/18/2024
 - DEP: Removed dependency of the `base-62` crate in the Rust codebase, since it depended on the `failure` crate which has a known [vulnerability](https://github.com/advisories/GHSA-jq66-xh47-j9f3).
 - BUG: Fix unhandled exception raised by `CommonAnnotatedKey.TryCreate(string, out CommonAnnotatedKey)` when passed non-CASK secrets of length < 80.
-- BUG: Update `AzureEventGridIdentifiableKey` rule id to `SEC101/199` to be synced with source of the rule.
+- BUG: Correct `AzureEventGridIdentifiableKey` rule id from `SEC101/190` to `SEC101/199` to be synced with source of the rule.
```

Comment:
>Correct [](http://example.com/codeflow?start=7&length=7)

I corrected the rule id elsewhere in release history where it previously referred to the incorrect version. And then I updated this release note to make it explicit that `SEC101/190` was a bogus (now fixed) rule id.